### PR TITLE
wip: add aria-label on social icons

### LINKED ISSUE #257: Improve Accessibility of Social Media Links
The links to external sites such as Twitter require a text label. Currently the links have the local site's title instead of the external site's title. For example, the Twitter link should read "Twitter"

### PR DESCRIPTION
**Pull request by jekyllbot**
Fix #257 

```text
fix: add aria-label on social icons
```

### CODE REVIEW

File: _includes/social.html [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-  {%- if social.telegram -%}<li><a rel="me" href="https://t.me/{{ social.telegram | cgi_escape | escape }}" title="{{ social.telegram | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#telegram' | relative_url }}"></use></svg></a></li>{%- endif -%}
-  {%- if social.keybase -%}<li><a rel="me" href="https://keybase.io/{{ social.keybase | cgi_escape | escape }}" title="{{ social.keybase | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#keybase' | relative_url }}"></use></svg></a></li>{%- endif -%}
-  {%- if social.microdotblog -%}<li><a rel="me" href="https://micro.blog/{{ social.microdotblog | cgi_escape | escape }}" title="{{ social.microdotblog | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#microdotblog' | relative_url }}"></use></svg></a></li>{%- endif -%}
-  {%- if social.rss -%}<li><a href="{{ 'feed.xml' | relative_url }}" title="{{ social.rss | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#rss' | relative_url }}"></use></svg></a></li>{%- endif -%}
```
Comment (ashmaroli) [inline _includes/social.html]: The `title` attribute would render a popup onHover..

File: _includes/social.html [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
+  {%- if social.telegram -%}<li><a rel="me" href="https://t.me/{{ social.telegram | cgi_escape | escape }}" aria-label="{{ social.telegram | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#telegram' | relative_url }}"></use></svg></a></li>{%- endif -%}
+  {%- if social.keybase -%}<li><a rel="me" href="https://keybase.io/{{ social.keybase | cgi_escape | escape }}" aria-label="{{ social.keybase | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#keybase' | relative_url }}"></use></svg></a></li>{%- endif -%}
+  {%- if social.microdotblog -%}<li><a rel="me" href="https://micro.blog/{{ social.microdotblog | cgi_escape | escape }}" aria-label="{{ social.microdotblog | escape }}"><svg class="svg-icon grey"><use xlink:href="{{ '/assets/minima-social-icons.svg#microdotblog' | relative_url }}"></use></svg></a></li>{%- endif -%}
+  {%- if social.rss -%}<li><a href="{{ 'feed.xml' | relative_url }}" aria-label="{{ social.rss | escape }}"><svg class="svg-icon grey" aria-label="RSS Feed"><use xlink:href="{{ '/assets/minima-social-icons.svg#rss' | relative_url }}"></use></svg></a></li>{%- endif -%}
```
Comment (ashmaroli) [inline _includes/social.html]: Why two `aria-label` on just this icon..?

File: _includes/social.html (hunk 1 above)
Comment (DirtyF) [reply _includes/social.html]: I know I'm following a11ty recommendations here; https://www.w3.org/TR/WCAG20-TECHS/ARIA8.html

Comment (ashmaroli) [reply _includes/social.html]: Perhaps I'm not seeing it, but where does it say the `title` attribute is not to be used in conjunction with `aria-label`..?

Comment (DirtyF) [reply _includes/social.html]: Screens readers tend to favor one over the other. From a pure a11ty title is not needed.

Comment (ashmaroli) [reply _includes/social.html]: okay then..

**Comment by matm78** [comment]
Markyz Rwzalys

**Pull request closed without merging**